### PR TITLE
from yujun:add new requirement.md

### PR DESCRIPTION
**Pull request by sunhao2017**
from yujun:add new requirement.md

### CODE REVIEW

**Comment by queenypingcap** [comment]
LGTM.

File: op-guide/requirement.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+| Oracle Enterprise Linux * | 7.3 及以上 |
+| Ubuntu LTS | 14.04 及以上 |
+
+**注：只支持 Red Hat 兼容内核 (RHCK) ,  不支持 Oracle Enterprise Linux 提供的 Unbreakable Enterprise Kernel。**
+
+**注 ：以上所支持的 Linux 操作系统包括部署和运行在物理服务器以及 VMWare, KVM, XEN 主流虚拟化环境。**
+
+## 服务器要求
+
+TiDB 支持部署和运行在 Intel  x86-64 架构的 64位通用硬件服务器平台。对于开发，测试，及生产环境的服务器硬件配置有以下要求和建议：
```
Comment (andelf) [inline op-guide/requirement.md]: redundant space here

File: op-guide/requirement.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+* **开发及测试环境**
+
+| 组件 | CPU | 内存 | 本地存储 | 网络 | 数量(最低要求) |
+| :-- | :-: | :-: | :-: | :-: | :-: |
+| TiDB| 16核+ | 32 GB+| SAS, 200 GB+ |千兆网卡 | 1|
+| TiKV| 16核+ | 32 GB+| SAS, 200 GB+ | 千兆网卡| 1|
+| PD | 8核+| 16 GB+| SAS, 200 GB+| 千兆网卡|3 |
+|||||服务器总计|5|
+
+**注：验证测试环境中的TiDB和PD 可以部署在同一台服务器上。**
```
Comment (andelf) [inline op-guide/requirement.md]: spaces

File: op-guide/requirement.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+**注：如进行性能相关的测试，避免采用低性能存储和网络硬件配置，防止对测试结果的正确性产生干扰。**
+
+* **生产环境**
```
Comment (andelf) [inline op-guide/requirement.md]: use `###` for level-3 title

**Comment by queenypingcap** [comment]
@buggithubs PTAL.

**Pull request merged**